news.Rmd

```diff
@@ -2,12 +2,37 @@
 
 Each user-facing change to a package should be accompanied by a bullet in `NEWS.md`. Minor changes to documentation don't need to be documented, but it's worthwhile to draw attention to sweeping changes and to new vignettes.
 
-## Bullets
+## In-development
 
 The goal of the bullet is to briefly describe the change so users of the packages can understand what's changed. This can be similar to the commit message, but written with a user (not developer) in mind.
 
-New bullets should be added to the top of the file (under the first heading).
-Organisation of bullets will happen later, during the release process (Section \@ref(news-release)).
+New bullets should be added to the top of the file (immediatley under the first heading) and should be a single line (possibly wrapped). Organisation and wrapping will happen later, during the release process (Section \@ref(news-release)).
+
+```
+# haven (development version)
+
+* Second update.
+* First update.
+```
+
+If the bullet is related to an issue, include the issue number. If the
+contribution is a PR, and the author is not a package author, include the
+GitHub user name. Both items should be wrapped in parentheses and will generally
+come before the final period.
+
+```
+# Good
+* `ggsave()` now uses full argument names to avoid partial match warnings (@wch, #2355).
+
+# Bad
+* `ggsave()` now uses full argument names to avoid partial match warnings.
+
+* `ggsave()` now uses full argument names to avoid partial match warnings. (@wch, #2355)
+```
+
+## Pre-release {#news-release}
+
+Prior to release, the NEWS file needs to be thoroughly proofread, groomed, and organised into sections.
```

Comment:
Do you want to explicitly mention line wrapping here since you referred to this section from above? Are we planning on "manually" line wrapping the NEWS file before the release? Should we specify a character limit?

Reply:
I have been manually wrapping, but I'm not sure it's that important?